xml/System.Runtime.Intrinsics.X86/Sse2.xml

```diff
@@ -56,10 +54,7 @@
       <Docs>
         <param name="left">To be added.</param>
         <param name="right">To be added.</param>
-        <summary>
-            __m128i _mm_add_epi8 (__m128i a,  __m128i b)
-              PADDB xmm, xmm/m128
-            </summary>
+        <summary>__m128i _mm_add_epi8 (__m128i a,  __m128i b)<br />&#160;&#160;&#160;&#160;PADDB xmm, xmm/m128</summary>
```

Comment:
I'm going to test `&nbsp;` here, @rpetrusha.
```suggestion
        <summary>__m128i _mm_add_epi8 (__m128i a,  __m128i b)<br />&nbsp;&nbsp;&nbsp;&nbsp;PADDB xmm, xmm/m128</summary>
```

Reply:
Spaces will be ignored. Can we say instead Instruction: on the beginning of the 2nd line?

Reply:
Also, I'd use `<para>` instead of `<br>` tags to make sure this will render as expected in VS. We'd have to test how <br> tags would be converted in the IntelliSense files and in VS.